webapp/portlet/src/main/webapp/getting-started/components/ExoGettingStarted.vue

```diff
@@ -105,6 +108,19 @@ export default {
           this.showGettingStrated = false;
         }
       });
+    },
+    clearCache(){
```

Comment:
No need to iterate over all c aches, you can get the on,e you need and delete a specific entry inside .
check the example in  https://developer.mozilla.org/en-US/docs/Web/API/Cache/delete
`caches.open('portal-pwa-resources-dom').then(function(cache) {
  cache.delete('/dom-cache?id=GettingStartedPortlet').then(function(response) {
    console.debug('cache deleted');
  });
})`

Reply:
And provide a description for the Pull Request

Reply:
ok

Reply:
@Rdidmohamedelhadi I understand that people who will click on the close button will have the widget disappear, what about people who had already clicked on the close button ?